Core.hpp

```diff
@@ -6,24 +6,30 @@
 #define ABSTRACT_VM_CORE_H
 
 #include <iostream>
-#include <vector>
+#include <list>
 #include <cstring>
+#include "Instruction.hpp"
+#include "InstructionFactory.hpp"
 
 class Core {
 private:
-    enum errors {INVALID_INPUT, INVALID_COMMAND, INVALID_TYPE, INVALID_VALUE};
-    std::vector<errors> errorlist;
+    enum                        errors {INVALID_INPUT, INVALID_COMMAND, INVALID_TYPE, INVALID_VALUE};
```

Comment:
This is ridiculous. Please change this setting, because multiple spaces\tabs are there to make the code readable.